tools/exporters/data_dictionary_builder.php

```diff
@@ -36,100 +36,102 @@
  * Usage:
  *    php data_dictionary_builder.php
  *
- * PHP Version 5
+ * PHP Version 5,7
  *
  * @category Behavioural
  * @package  Main
  * @author   Loris Team <loris.mni@bic.mni.mcgill.ca>
  * @license  Loris License
  * @link     https://github.com/aces/Loris
  */
+require_once __DIR__ . '/../generic_includes.php';
 
-set_include_path(get_include_path().":../project/libraries:../php/libraries:");
+$parameter_types = array();
+$instrumentParameterTypeCategoryIDs = array();
+$instrumentParameterTypeIDs         = array();
 
-require_once __DIR__ . "/../vendor/autoload.php";
-$configFile = "../project/config.xml";
-$client     = new NDB_Client();
-$client->makeCommandLine();
-$client->initialize($configFile);
-$DB = Database::singleton();
 
-// Set a user for the history table
-if (!isset($_SESSION['State'])) {
```

Comment:
this is automatically done in the database class, no need for it here

Reply:
db class will get the current user vs here it's the owner of this script. this will break for any project that hasn't set up a matching db/system user.

Reply:
will consider it